components/provider_cldr/src/transform/dates.rs

```diff
@@ -161,10 +161,11 @@ impl From<&cldr_json::DateTimeFormats> for gregory::patterns::DateTimeFormatsV1
                     };
```

Comment:
Nit (optional): Go ahead and add en-CA to resources/testdata/Cargo.toml under the heading "Extra data for feature coverage in provider_cldr tests" and re-generate the data to include it.  This way we won't regress.

Reply:
Done, I force pushed and update because my first attempt was wrong (I added all the data).